### ISSUE TITLE
iOS build failing during "Upload Debug Symbols to Sentry"

### ISSUE THREAD

**Issue by ddoria921** [getsentry/sentry-cli#356]
I just added `sentry-cordova` to my project and tried to build my iOS app. During the build step I'm running into this error

```
# Reading property from /Users/Darin/Development/cordova-app/cordova/sentry.properties
/Users/Darin/Development/cordova-app/cordova/node_modules/@sentry/cli/bin/sentry-cli:6
const cli = require('../js');
^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

The node version on my machine is `v6.10.1` so I don't know why it would be failing with the use of `const`.

**Comment by mitsuhiko** [comment]
Actually was not hard to replicate once you point it to the right files. Fixed in 69005f01ceb59195485de0780d3c0ec0debbcc56

**Comment by mitsuhiko** [comment]
Responded to the wrong issue.
